pages/common/ocrmypdf.md

```diff
@@ -0,0 +1,28 @@
+# ocrmypdf
+
+> Generate a searchable PDF or PDF/A from a scanned PDF or an image of text.
+> More information: <https://ocrmypdf.readthedocs.io/en/latest/introduction.html>.
+
+- Create a new searchable PDF/A file from a scanned PDF or image file:
+
+`ocrmypdf {{input_file}} {{output.pdf}}`
+
+- Overwrite a scanned PDF file with a searchable PDF file, if successful:
+
+`ocrmypdf {{input.pdf}}`
+
+- Skip pages of a mixed-format input PDF file that already contain text:
+
+`ocrmypdf --skip-text {{input.pdf}} {{output.pdf}}`
+
+- Automatically clean, de-skew and rotate pages of a poor scan:
```

Comment:
```suggestion
- Clean, de-skew and rotate pages of a poor scan:
```
I wouldn't call it automatic.

Reply:
"Automatically" was from the author description. I think its superfluous too. Oxford comma?
https://youtu.be/SJ-3PJkcOTo

Reply:
It's important to speak correctly.

However don't compare me to any kind of Nazi. :)

Reply:
I'm glad that you recognized it as a joke. Critiquing an Oxford comma in the setting of a review IS appropriate.